gatsby-node.js

```diff
@@ -21,22 +23,29 @@ exports.createPages = async ({ actions, graphql, reporter }) => {
                   frontmatter {
                     slug
                   }
+                  parent {
```

Comment:
That's exactly what it does. Each node returned by the `allMarkdownRemark` query has a `parent` field, that is an instance of a `File` node.

Reply:
Actually, it doesn't give me the file name; it allows me to check which instance of `gatsby-source-filesystem` the `markdownRemark` node comes from. This is probably left over from my first attempts of rendering the stdlib. I should probably get rid of it, since we're now using the JSON documentation export and rendering the markdown manually.